content/applications/services/helpdesk/overview/getting_started.rst

```diff
@@ -2,63 +2,180 @@
+
+.. important::
+   :doc:`Developer mode</applications/general/developer_mode>` must be activated in order to access
+   the stages menu. To activate developer mode go to
+   :menuselection:`Settings --> General Settings --> Developer Tools` and click on
+   :guilabel:`Activate the developer mode`.
+
+The list view shows an overview of all the stages currently available in Helpdesk. They are listed
+in the order they appear in the pipeline. To change the order of the stages, use the arrow buttons
+on the left side of the list.
+
+.. tip::
+   Change the stage order on the kanban view by dragging and dropping individual columns.
+
+.. image:: getting_started/stages-create-new.png
+   :align: center
+   :alt: View of the stage list page emphasizing the option to create a new stage
+
+
+To create a new stage, click on the :guilabel:`New` button in the top left of the dashboard.
+Next, choose a name for the new stage, and add a description (though it is not required).
+Fill out the remaining fields following the steps below.
+
+.. image:: getting_started/stage-settings.png
+   :align: center
+   :alt: View of a stage's settings page in Odoo Helpdesk
+
+
+Add email and SMS templates to stages
+-------------------------------------
+
+When an :guilabel:`Email Template` is added to a stage, an email is automatically sent to the
+the customer when a ticket reaches the stage. An :guilabel:`SMS Template` will result in an
+SMS text message sent to the customer.
+
+.. important::
+   SMS Text Messaging is an In-App Purchase (IAP) service that requires prepaid credits to work.
+   Refer to :doc:`SMS Pricing and FAQ
+   </applications/marketing/sms_marketing/pricing/pricing_and_faq>`
+   for additional information.
+
+
+To select an existing email template, select it from the :guilabel:`Email Template` field. Click on
+the arrow key to the right of the field to edit the template.
+
+To create a new template, click the field and begin typing a new template title. Then select
+:guilabel:`Create and edit`.
+
+Follow the same steps to select, edit, or create an :guilabel:`SMS Template`.
+
+.. image:: getting_started/sms-template.png
+   :align: center
+   :alt: View of an SMS template setup page in Odoo Helpdesk
+
+
+Assigning stages to a team
+--------------------------
+
+Make a selection in the :guilabel:`Teams` field. More than one team may be selected, as stages
+can be shared among multiple teams.
+
+ .. image:: getting_started/stages-settings-sharing.png
+   :align: center
+   :alt: View of stage setup emphasizing teams field
+
+.. example::
+   In the setup below, the stages `New`, `In Progress`, and `Canceled` are shared between the
+   `Customer Care` and `VIP Support` teams. The other teams are only available to one of the two
+   teams.
+
+  .. image:: getting_started/stages-sharing.png
+   :align: center
+   :alt: View of stages list in Helpdesk showing shared stages
```

Comment:
Redundant / don't need.
Also, images are supplemental to the text, and the text should stand on it's own to provide instruction; text should not rely on images to convey important information.
```suggestion
```